.changeset/red-snakes-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-explore': patch
+---
+
+Updated the dev app to work properly
```

Comment:
No need for a changeset here really, or we can just mentioned that dev deps were updated

Reply:
removed!